source/_components/input_select.markdown

```diff
@@ -79,3 +79,49 @@ scene:
       input_select.who_cooks:
         option: Paulus
 ```
+
+Exampleof `input_select` being used in a bidirectional manner, both being set by and controlled by an MQTT action in an automation.
```

Comment:
Ah, I copied/pasted from input_slider. This was my first pull request, so do I somehow modify this request or start a new one?